client/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<link rel="stylesheet" type="text/css" href="index.css">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<meta charset="utf-8">
+		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+		<title> 
+				Code Runner
+		</title> 
+		<link rel = "icon" href = "images/logo.png"  type = "image/x-icon">
+		<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+
+		<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+	</head>
+	<body>
+	<div class="container top-container">
+			 <div class="header" id="app-header">
+			 <div class="side-nav sn-bg-4 mdb-sidenav fixed">
+				<img src="images/logo.png" alt="logo" height="40" class="pull-left heading" />
+				<h1>Code Runner</h1>
+			</div>
+			</div>
+			<div class="content"> 
+			<form id="myForm">
```

Comment:
follow the same naming convention. Seems like both hyphenation and camelcasing are being used.

Reply:
Okay